doc/api/vm.md

```diff
@@ -9,8 +9,10 @@
 <!-- source_link=lib/vm.js -->
 
 The `vm` module enables compiling and running code within V8 Virtual
-Machine contexts. **The `vm` module is not a security mechanism. Do
-not use it to run untrusted code.**
+Machine contexts.
+
+<span class="critical">The `vm` module is not a security
+mechanism. Do not use it to run untrusted code.</span>
```

Comment:
Semantic markup ftw
```suggestion
<strong class="critical">The `vm` module is not a security
mechanism. Do not use it to run untrusted code.</strong>
```

Reply:
This would be a rare instance of using `<strong>` semantically rather than using `<strong>` instead of `<b>` and pretending that it's semantic.